jobs/config.json

```diff
@@ -2634,6 +2634,7 @@
       "--gcp-project-type=gci-qa-project",
       "--gcp-zone=us-central1-f",
       "--provider=gce",
```

Comment:
Hmm, why not just `[Serial]` here? Unless the gce equivalent running with the same `test_args`?

Reply:
if you look at ci-kubernetes-e2e-gce-gci-qa-serial-m63 above, that has the right focus :-)

Reply:
@wonderfly you can check the config file, but it's pretty standard for `*-serial-*` suite to run these set of tests. There is no reason for this to be different.

Reply:
Sorry I have fallen behind the config changes. Thanks for catching this. Hopefully it'll be a fix for b/69273090 as well. :)